src/shims/shims.proj

```diff
@@ -43,31 +42,23 @@
 
     <Exec Command="$(GenFacadesCmd) -contracts:&quot;@(NETStandardContracts)&quot; @&quot;$(GenFacadesResponseFile)&quot;" WorkingDirectory="$(ToolRuntimePath)" />
 
-    <!-- Copy the facades to the package ref and lib folders to be included in the packages -->
-    <!-- TODO: replace with BinPlacing targets -->
-    <ItemGroup>
-      <PackageOutputPaths Include="$(BinDir)pkg/$(TargetGroup)/ref" Condition="'$(_bc_TargetGroup)'!='uapaot'"/>
-      <PackageOutputPaths Include="$(BinDir)pkg/uap/ref" Condition="'$(_bc_TargetGroup)'=='uapaot'"/>
-      <PackageOutputPaths Include="$(BinDir)pkg/$(TargetGroup)/lib" />
-      <PackageOutputPaths Include="$(RefPath)" />
-      <PackageOutputPaths Include="$(RuntimePath)" />
```

Comment:
Will they still get deployed into the RuntimePath? From what I can tell, it looks like they will only get placed into the `bin\pkg` dirs now.